src/Source/AbstractSource.php

```diff
@@ -133,11 +136,11 @@ protected function deleteValueFromRemoteCache(string $cacheKey): string
         return $filesystemCache->deleteItem('entity-import-remote.'.$cacheKey);
     }
 
-    protected function storeValueToRemoteCache(string $url, string $cacheKey, string $method, array $auth = [])
+    protected function storeValueToRemoteCache(string $url, string $cacheKey, string $method, array $auth = [], array $HTTPClientOptions = [])
```

Comment:
What about merging $auth and your new array to an extensible $httpClientOptions array (watch code style)? I know it's not 100% backwards compatible but the current solution is not really extensible. Adding $auth was not general enough in my opinion. It should be part of a general client options array. As this bundle is still in beta stage, this won't be a problem.

Reply:
I don't think these are the same, because referring to the Guzzle docs $auth is a request parameter, while SSL checking is an option for the Client at setup time. Maybe we could agree on an array of the form ['headers' => [$auth], 'options' => [$httpClientOptions]]?

Reply:
I know they're not the same ;-) I meant something like

```
$options = [
  'auth' => ...,
   // ...
]
```

Such an array we could pass to

```
$client->request('GET', 'https://api.github.com/user', [
    'auth' => ['user', 'pass']
]);

// -->

$client->request('GET', 'https://api.github.com/user', $options);
```

OK?

Reply:
So not passing the options to the client's constructor but the request() method.

Reply:
Do you mean two arrays in the function definition? One for the constructor and one for the request? Or if not, where do I put the "verify" part? Sorry if I am slow on the uptake... Oh, sry, I think now I've got it, what you mean... you would pass the verify to the request too... before looking into the Guzzle-docs (again) I did not know, that this was possible... Is that what you've meant?

Reply:
Yep. Only one parameter ($options) for the method which we then pass to the request() method. This way we are way more extensible.

Reply:
I pushed an update in 65e7485. Did you mean it like that?